src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -1799,6 +1801,7 @@ static void gfx_calc_and_set_viewport(const Vp_t* viewport) {
 }
 
 static void gfx_sp_movemem_f3dex2(uint8_t index, uint8_t offset, const void* data) {
+#ifdef F3DEX_GBI_2
```

Comment:
There should not be any gbi specific ifdefs or checks inside of handlers. We should split this into two functions, one for f3dex2 and one for f3dex.